homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -312,22 +312,16 @@ def __init__(self, name, device, entry, unique_id, coordinator):
         """Initialize the generic Xiaomi device."""
         super().__init__(name, device, entry, unique_id, coordinator)
 
-        self._available = False
+        self._attr_available = False
         self._available_attributes = {}
-        self._state = None
         self._mode = None
         self._fan_level = None
-        self._state_attrs = {ATTR_MODEL: self._model}
+        self._attr_extra_state_attributes = {ATTR_MODEL: self._model}
```

Comment:
Do we really need `model` attribute?
User can check the `model` in the frontend:

![image](https://user-images.githubusercontent.com/478555/130348066-1a939242-17c9-40de-8e09-6635a760de99.png)

Reply:
We should remove it (in a follow up).